exercises/exercise_000_sudoku_solver_initial_state/src/main/scala/org/lunatechlabs/dotty/sudoku/SudokuProgressTracker.scala

```diff
@@ -40,7 +40,8 @@ class SudokuProgressTracker private (
       case NewUpdatesInFlight(updateCount) =>
         trackProgress(updatesInFlight + updateCount)
       case msg: SudokuDetailState =>
-        context.log.error("Received unexpected message in state 'trackProgress': {}", msg)
+        // context.log.error("Received unexpected message in state 'trackProgress': {}", msg)
```

Comment:
Maybe we can add an issue in this repo that is a TODO to go back and fix this when the dotty issue (https://github.com/lampepfl/dotty/issues/9688) is addressed? And the issue in our course repo can link the issue in the dotty repo. And then in the code you can add a link to the new issue (otherwise the commented out code isn't really useful by itself)

Reply:
Good call!

Will do as otherwise, there's the risk that we forget to revert the workaround when it's no longer needed.